core_helpers/logging_objects.py

```diff
@@ -1,16 +1,24 @@
 """
 Helper class for Logging Objects
 """
-
+import os,inspect
 from utils.Base_Logging import Base_Logging
 from utils.stop_test_exception_util import Stop_Test_Exception
+from utils import Gif_Maker
 import logging
 
-class Logging_Objects:
-    def __init__(self):
+class Logging_Objects():
+    def __init__(self, base_url=None):
```

Comment:
Do we need `base_url` in `Logging_Objects`?

Reply:
It's not needed @shivahari, thanks. Removed it.